index.html

```diff
@@ -117,5 +117,40 @@ <h3>how we started</h3>
         </ul>
       </div>
     </section>
+    <section class="contact">
+      <h2>contact</h2>
+      <form>
+        <label for="name">name</label>
+        <input type="text" id="name">
+        <label for="email">e-mail</label>
+        <input type="email" id="email">
+        <label for="topic">topic</label>
+        <textarea id="topic"></textarea>
+      </form>
+      <img src="assets/office.jpg" alt="Hello agency headquaters">
+      <div>
+        <span>Our office is located in the heart of Warsaw</span>
+        <span>Feel free to say hello!</span>
+        <div itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
```

Comment:
Should I also have a `itemprop="name"` somewhere and a whole `itemtype` of `Place` type?